src/components/Navbar.js

```diff
@@ -1,20 +1,27 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { CgProfile } from 'react-icons/cg';
 import styles from '../styles/Navbar.module.css';
 
 const Navbar = () => (
-  <div className={styles.navbar}>
-    <h1>BookStore </h1>
-    <ul className={styles.nav}>
-      <li>
-        <Link to="/">Home</Link>
-      </li>
-      <div className={styles.border} />
-      <li>
-        <Link to="/categories">Categories</Link>
-      </li>
-    </ul>
-  </div>
+  <>
+    <nav className={styles.nav}>
+      <div className={styles.title}>
+        <h1>BookStore CMS</h1>
+        <ul className={styles.navlinks}>
+          <li className={styles.links}>
+            <Link to="/">Books</Link>
+          </li>
+          <li className={styles.linksCat}>
+            <Link to="/categories">Categories</Link>
+          </li>
+        </ul>
+      </div>
+      <div className={styles.profile}>
```

Comment:
Well done by adding the profile icon but please do your best to make it look like the design in terms of size and color 

![image (21)](https://github.com/ragangithub/bookstore-react/assets/59707859/fcb93711-23fd-4219-8095-184284ec3ebc)

![image (26)](https://github.com/ragangithub/bookstore-react/assets/59707859/b4829961-15ca-4103-b214-7b485cfe0791)